jboss-eap-cd-openshift-launch/added/launch/messaging.sh

```diff
@@ -105,10 +118,26 @@ function configure_mq() {
     configure_mq_cluster_password
 
     destinations=$(configure_mq_destinations)
-    activemq_subsystem=$(sed -e "s|<!-- ##DESTINATIONS## -->|${destinations}|" <"${ACTIVEMQ_SUBSYSTEM_FILE}" | sed ':a;N;$!ba;s|\n|\\n|g')
+    
+    # We need the broker if they configured destinations or didn't explicitly disable the broker AND there's a point to doing it because the marker exists
+    if ([ -n "${destinations}" ] || [ "x${DISABLE_EMBEDDED_JMS_BROKER}" != "xtrue" ]) && grep -q '<!-- ##MESSAGING_SUBSYSTEM_CONFIG## -->' ${CONFIG_FILE}; then
+
+      log_warning "Configuration of an embedded messaging broker within the appserver is enabled but is not recommended. Support for such a configuration will be removed in a future release."
+      if [ "x${DISABLE_EMBEDDED_JMS_BROKER}" != "xtrue" ]; then
+        log_info "If you are not configuring messaging destinations, to disable configuring an embedded messaging broker set the DISABLE_EMBEDDED_JMS_BROKER environment variable to true."
```

Comment:
Bit of a grammar tweak, maybe?

Reply:
Actually, never mind, I just read it wrong the first time :)

Reply:
Not saying this is better ;)

If you are not configuring messaging destinations, and wish to disable the configuration of the embedded messaging broker; set the DISABLE_EMBEDDED_JMS_BROKER environment variable to true.